nats/bench_command.go

```diff
@@ -49,10 +62,17 @@ func configureBenchCommand(app *kingpin.Application) {
 	bench.Flag("size", "Size of the test messages").Default("128").IntVar(&c.msgSize)
 	bench.Flag("csv", "Save benchmark data to CSV file").StringVar(&c.csvFile)
 	bench.Flag("progress", "Enable progress bar while publishing").Default("true").BoolVar(&c.progress)
-	bench.Flag("ack", "Waits for acknowledgement on messages using Requests rather than Publish").Default("false").BoolVar(&c.ack)
+	bench.Flag("request", "Waits for acknowledgement on messages using Requests rather than Publish").Default("false").BoolVar(&c.request)
+	bench.Flag("streaming", "Use JetStream streaming").Default("false").BoolVar(&c.js)
```

Comment:
maybe `-js` like in the server?

Reply:
That's how I had it initially, but then thought that the new messaging is that JS is the new persistence layer of NATS and that now streaming is implemented with JS ... but there are also other things (e.g. kv store) on top of JS as well as streaming, so I changed it to --streaming (and the help says 'Use JetStream streaming'). But open to changing it back to --js if using --streaming feels wrong to you.

Reply:
Makes sense though I'd leave it as `-js` tot be consistent with the server for now since nats streaming has a couple of years still to be deprecated.

Reply:
reverted back to --js